src/IyyCrv.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.4.21 <0.7.0;
+
+// Because USDT is not so standard ERC20, we just use their code as interface
```

Comment:
don't use USDT's interface, it's **NON STANDARD**

![image](https://user-images.githubusercontent.com/3261732/92246891-f8b2f800-eef8-11ea-8fcf-9f95e0b25449.png)